src/jit/namedintrinsiclist.h

```diff
@@ -49,7 +49,7 @@ enum HWIntrinsicFlag : unsigned int
     HW_Flag_Generic = 0x4,
```

Comment:
I think renaming this to be `HW_Flag_OneTypeGeneric` would be the best route, but don't feel particularly strong about it.